cdm/core/src/main/java/ucar/nc2/filter/Standardizer.java

```diff
@@ -0,0 +1,80 @@
+package ucar.nc2.filter;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.commons.math.stat.descriptive.SummaryStatistics;
+import ucar.ma2.Array;
+import ucar.ma2.DataType;
+import ucar.ma2.IndexIterator;
+import ucar.nc2.dataset.VariableDS;
+
+public class Standardizer {
+
+  private final ScaleOffset scaleOffset;
+  private static final String name = "Standardizer";
+  private static final int id = -1;
```

Comment:
```suggestion
```
We don't use these anymore now that the `Standardizer` isn't a `Filter`

Reply:
Both lines were removed.